### PR TITLE
[Cherry-Pick] Reunion VSIX: Update title of "Blank App, Packaged with WAP" project type (#1622)

### PR DESCRIPTION
**Pull request by ryalanms**
Update the Visual Studio template title to "Blank App, Packaged with Windows Packaging Project (WinUI 3 in Desktop)" to avoid using "WAP". (Tracked by an internal bug.)

The change and associated localization commits were approved by WinAppSdk shiproom.

### CODE REVIEW

File: dev/VSIX/Extension/Cpp/Common/de-DE/VSPackage.de-DE.resx [hunk 1]
```diff
@@ -185,11 +185,8 @@
   <data name="1023" xml:space="preserve">
     <value>Ein Benutzersteuerelement für Apps, die auf der Windows-UI-Bibliothek (WinUI 3) basieren.</value>
   </data>
-  <data name="1025" xml:space="preserve">
```
Comment (ryalanms) [inline dev/VSIX/Extension/Cpp/Common/de-DE/VSPackage.de-DE.resx:188]: This translation is missing.

File: dev/VSIX/Extension/Cpp/Common/fr-FR/VSPackage.fr-FR.resx [hunk 2]
```diff
@@ -185,12 +185,6 @@
   <data name="1023" xml:space="preserve">
     <value>Contrôle utilisateur pour les applications basé sur la Bibliothèque d’interface utilisateur Windows (WinUI 3).</value>
   </data>
-  <data name="1025" xml:space="preserve">
```
Comment (ryalanms) [inline dev/VSIX/Extension/Cpp/Common/fr-FR/VSPackage.fr-FR.resx:188]: The French translations are also missing.

**Review by Scottj1s — APPROVED** [review]

**Comment by ryalanms** [comment]
Thank you, @DefaultRyan and @Scottj1s.

**Comment by ryalanms** [comment]
@DefaultRyan:  This still requires an authorized reviewer.  Could you sign-off?

**Pull request merged**